tweet_bible_verse.py

```diff
@@ -40,8 +40,8 @@ def get_bible_verse():
     response.raise_for_status()
 
     soup = BeautifulSoup(response.content, "html.parser")
-    verse = soup.select(".votd-box > p")[0].text
-    verse_url = urljoin(BIBLE_VERSE_URL, soup.select(".votd-box > a")[0]['href'])
+    verse = soup.find_all(id="verse-text")[0].text
+    verse_url = urljoin(BIBLE_VERSE_URL, soup.find_all("div", "verse-bar")[0].contents[1]["href"])
```

Comment:
I think you can simplify this a bit more for readability's sake if you use "soup.select_one". Also, could you also come up with a simple css selector for accessing the URL -- You can directly access the .verse-bar css class and then proceed.